--- libbtbb-2020-12-R1/python/pcaptools/CMakeLists.txt.orig	2020-12-25 22:43:09.000000000 +0100
+++ libbtbb-2020-12-R1/python/pcaptools/CMakeLists.txt	2022-07-05 16:08:35.481831720 +0200
@@ -31,7 +31,7 @@ add_custom_command(OUTPUT ${OUTPUT}/time
                    DEPENDS ${DEPS})
 
 add_custom_target(pcapdump ALL DEPENDS ${OUTPUT}/timestamp)
-install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} build -b ${OUTPUT} install --prefix=${BUILD_ROOT}/${CMAKE_INSTALL_PREFIX})")
+install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} build -b ${OUTPUT} install --skip-build --prefix=${CMAKE_INSTALL_PREFIX} --optimize=2 --root=\$ENV{DESTDIR})")
 
 add_custom_target(btaptap ALL DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/btaptap)
 install(PROGRAMS btaptap DESTINATION ${INSTALL_DEFAULT_BINDIR})
